signal/signals.go

```diff
@@ -82,3 +85,8 @@ func NotifyNode(jsonEvent *C.char) {
 func TriggerTestSignal() {
 	C.StatusServiceSignalEvent(C.CString(`{"answer": 42}`))
 }
+
+// SetSignalEventCallback set callback
+func SetSignalEventCallback(cb unsafe.Pointer) {
```

Comment:
@adambabik thank you for the correction, have updated functions naming.

```
s/void* cb/void *cb to keep consistency.

Also, this is not related to jail. Signals are sent from different parts of the app. SetSignalEventCallback(void *cb) sounds more appropriate.
```

Could you please elaborate regarding `s/void* cb/void *cb to keep consistency.` Not sure that I'm getting this correctly.

Also the question regarding naming best practice:
Go function `SetSignalEventCallback` finally calls C function `SetEventCallback` defined in `signals.c`. I would like to name C function identically to function defined in Go (since the purpose of both is the same), but compilation fails if I do that way. I'm wondering if exists some naming convention to point that function is exported from C in title of that function.

Thanks

Reply:
> `s/void* cb/void *cb` to keep consistency

It's just cosmetics. The star is put next to the argument name in this file so `void SetEventCallback(void *cb)` :)

I don't think we use any convention for that. I assume the compilation problem is because Go code is compiled as CGO and it basically shares the same symbol table. `void SetEventCallback(void *cb)` looks good to me though. I would keep it.

Reply:
@adambabik got it, thanks for clarification! Sorry, haven't understood `s/` correctly